packages/terra-toolkit-docs/src/terra-dev-site/tool/stylelintConfigTerra/UpgradeGuide.y.tool.mdx

```diff
@@ -4,7 +4,7 @@ import { Badge } from '@cerner/stylelint-config-terra/package.json?dev-site-pack
 
 # stylelint-config-terra Upgrade Guide
 
-## Changes from stylelint-config-terra 4.0.0  to @cerner/stylelint-config-terra 5.0.0
+## Changes from @cerner/stylelint-config-terra 4.0.0  to @cerner/stylelint-config-terra 5.0.0
```

Comment:
non-blocking nit: Just because we're editting this line, we could probably get rid of the double-space before the `to`

Reply:
Good catch, updated here: f5bc8ee310ad417e17bc943eaeac0b509a3801da